promql/functions.go

```diff
@@ -814,11 +863,11 @@ var functions = map[string]*Function{
 		Call:       funcResets,
 	},
 	"round": {
-		Name:         "round",
-		ArgTypes:     []model.ValueType{model.ValVector, model.ValScalar},
-		OptionalArgs: 1,
-		ReturnType:   model.ValVector,
-		Call:         funcRound,
+		Name:       "round",
+		ArgTypes:   []model.ValueType{model.ValVector, model.ValScalar},
+		Variadic:   true,
```

Comment:
Shouldn't we also keep optional args around in addition to variadic ones? Otherwise people can now pass an arbitrary number of arguments into functions like `round` or `delta`. /cc @fabxc @brian-brazil